travispy/entities/_entity.py

```diff
@@ -192,8 +192,7 @@ def _load(cls, infos, session):
                 try:
                     setattr(entity, key, value)
                 except AttributeError:
-                    warn('Unknown {0} attribute {1}'
-                         .format(entity.__class__.__name__, key))
+                    pass
```

Comment:
So by default end users should be warned about something that's nothing to do with them,  and need to change the warning level? Won't that then mean they would miss warnings in their own code that they should see?